CHANGELOG.md

```diff
@@ -3,3 +3,7 @@
 ## 1.0.0 (October 29, 2019)
 
 Initial release! :tada:
+
+## 1.0.1 (November 6, 2019)
```

Comment:
This entry should actually go above `1.0.0`.

Reply:
will do.